gpkit/tests/t_nomials.py

```diff
@@ -221,7 +224,10 @@ def test_eq_ne(self):
         xu = Variable('x', units="ft")
         with SignomialsEnabled():
             self.assertEqual(x - x**2, -x**2 + x)
-            self.assertNotEqual(-x, -xu)
+            if gpkit.units:
+                self.assertNotEqual(-x, -xu)
+            else:  # units don't create inequality if they're disabled
+                self.assertEqual(-x, -xu)
```

Comment:
good, I think we have consensus on removing that if you want to do it in the future, outside this PR. Not a huge priority but would roughly halve our unit test time which would speed CI noticeably!